README.md

```diff
@@ -43,6 +43,21 @@ aiidalab-launch status
 The tool allows to manage multiple profiles, e.g., with different home directories or ports.
 See `aiidalab-launch profiles --help` for more information.
 
+
+### Data Management
+By default AiiDalab will store all of its data in a [Docker volume](https://docs.docker.com/storage/volumes/)
+defined in the profile configuration option `home_mount`.
+You can also provide an absolute path to the AiiDalab home directory on the host system
+(so called [bind mount](https://docs.docker.com/storage/bind-mounts/)).
+If this directory does not exist, aiidalab-launch will try to create it on startup.
```

Comment:
```suggestion
If this directory does not exist, AiiDAlab launch will try to create it on startup.
```
"AiiDAlab launch" referring to the software by name
`aiidalab-launch` when referring to the executable 😉